proposals/null-conditional-assignment.md

```diff
@@ -151,12 +151,18 @@ class C
     ref int M() => /*...*/;
 }
 
-void M(C? c)
+void M1(C? c)
 {
     c?.M() = 42; // equivalent to:
     if (c is not null)
         c.M() = 42;
 }
+
+int? M2(C? c)
+{
+    return c?.M() = 42; // equivalent to:
+    return c == null ? (int?)null : c.M() = 42;
```

Comment:
Sometimes you use `is null`/`is not null`, other times `== null`/`!= null`, maybe consolidate those?

Reply:
Fixed, thanks!